Directory.Packages.props

```diff
@@ -0,0 +1,22 @@
+﻿<Project>
+	<ItemGroup>
+    	<PackageVersion  Include="BenchmarkDotNet" Version="0.13.1" />
+		<PackageVersion  Include="D2L.CodeStyle.Analyzers" Version="0.142.0" />
+		<PackageVersion  Include="D2L.CodeStyle.Annotations" Version="0.28.0" />
+		<PackageVersion  Include="D2L.Services.Core.Exceptions" Version="2.0.1.15" />
+		<PackageVersion  Include="FluentAssertions" Version="6.2.0" />
+		<PackageVersion  Include="Microsoft.AspNet.WebApi.Core" Version="[5.2.7]" />
+		<PackageVersion  Include="Microsoft.AspNet.WebApi.OwinSelfHost" Version="[5.2.7]" />
+		<PackageVersion  Include="Microsoft.Extensions.Caching.Memory" Version="5.0.0" />
+		<PackageVersion  Include="Microsoft.NET.Test.Sdk" Version="17.0.0" />
+		<PackageVersion  Include="Moq" Version="4.16.1" />
+		<PackageVersion  Include="Newtonsoft.Json" Version="[6.0.8,7)" />
```

Comment:
`6.*` may be slightly clearer, but no preference.

Reply:
Yeah, I'm fine with that but don't really care, mostly I'm nervous about upgrading past 6 when we haven't vetted it (this concern is specific to newtonsoft)

Reply:
Also, TBF, I doubt there will be another 6.0.x version at this point :D